homeassistant/components/intesishome/climate.py

```diff
@@ -181,11 +213,17 @@ def temperature_unit(self):
     def device_state_attributes(self):
         """Return the device specific state attributes."""
         attrs = {}
-        if len(self._swing_list) > 1:
-            attrs["vertical_vane"] = self._vvane
-            attrs["horizontal_vane"] = self._hvane
         if self._outdoor_temp:
             attrs["outdoor_temp"] = self._outdoor_temp
+        if self._power_consumption_heat:
+            attrs["power_consumption_heat_kw"] = round(
+                self._power_consumption_heat / 1000, 1
+            )
+        if self._power_consumption_cool:
+            attrs["power_consumption_cool_kw"] = round(
+                self._power_consumption_cool / 1000, 1
+            )
```

Comment:
Future: You can use with config flow the device API. So you can expose that as sensor and map it with the climate, look a lot better and is more usefully for future stuff.

Reply:
Thanks - yeah that's the long term plan just didn't want to include it in a feature update! Can you point me to a good example of this to look at?

Reply:
Here is a good tutorial: https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities